recipes/super_state_machine/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.0" %}
+
+package:
+  name: super_state_machine
+  version: {{ version }}
+
+source:
+  url: https://github.com/beregond/super_state_machine/archive/{{ version }}.tar.gz
+  fn: suitcase-{{ version }}.tar.gz
+  sha256: a4aac93307f381c64e25b24de347b0527608cc551b66f41c34b2f2df80b445b3
+  patches:
+    - nuke_enum34.patch  # [py3k]
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - enum34  # [py2k]
+    - six
+
+  run:
+    - python
+    - enum34  # [py2k]
+    - six
+
+test:
+  imports:
+    - super_state_machine
+
+about:
+  home: https://github.com/beregond/super_state_machine
+  license: BSD License
```

Comment:
@ericdill this seems to be BSD 3-Clause BTW. See https://github.com/beregond/super_state_machine/blob/master/LICENSE